File: RepoTransBench/target_projects/Python/C++/un33k_django-uuslug/public_tests/public_test_apps_py.cpp
#include <gtest/gtest.h>
#include <string>

namespace apps_mod {
    class UuslugConfig {
    public:
        std::string name;
        std::string verbose_name;
        UuslugConfig(const std::string& n, const std::string& vn)
            : name(n), verbose_name(vn) {}
    };
}

TEST(PublicTestAppsPy, test_apps_config_name_public) {
    apps_mod::UuslugConfig app_config("uuslug", "Uuslug package");
    ASSERT_EQ(app_config.name, "uuslug");
    ASSERT_TRUE(app_config.verbose_name.find("uuslug") != std::string::npos ||
                app_config.verbose_name.find("Uuslug") != std::string::npos);
}

File: RepoTransBench/target_projects/Python/Java/un33k_django-uuslug/src/test/java/com/example/public_tests/PublicTestAppsPy.java
package com.example.public_tests;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class AppsMod {
    static class UuslugConfig {
        String name;
        String verbose_name;
        UuslugConfig(String name, String label) {
            this.name = name;
            this.verbose_name = "Uuslug app";
        }
    }
}

public class PublicTestAppsPy {
    @Test
    void testAppsConfigNamePublic() {
        AppsMod.UuslugConfig appConfig = new AppsMod.UuslugConfig("uuslug", "uuslug");
        assertEquals("uuslug", appConfig.name);
        assertNotNull(appConfig.verbose_name);
        assertTrue(appConfig.verbose_name.toLowerCase().contains("uuslug"));
    }
}